keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -526,302 +517,382 @@ dualstack cluster can have this flag configured as
    example, an IPv4 cluster can safely be upgraded to use
    `--service-cluster-ip-range=<ipv4-cidr>,<ipv6-cidr>`, but can not safely be
    changed to `--service-cluster-ip-range=<ipv6-cidr>,<ipv4-cidr>`.
-5. The first CIDR entry of `--service-cluster-ip-range=<first cidr entry>,
-   <secondary cidr entry>` is considered cluster default IP address family.
+5. The first CIDR entry of `--service-cluster-ip-range` is considered cluster
+   default IP address family.
 6. Attempting to allocate `ClusterIP` or `ClusterIPs` from an IP address family
    that is not configured on the cluster will result in an error.
 
-> The below discussion assumes that `Endpoint Slice and Endpoint Slice
-> Controller`  will allow dual stack endpoints, the existing endpoint
+> The below discussion assumes that `EndpointSlice and EndpointSlice
+> Controller`  will allow dual stack endpoints, the existing Endpoints
 > controller will remain single stack. That means enabling dual stack services
 > by having more than one entry in `--service-cluster-ip-range` without
-> enabling `Endpoint Slice` feature gate will result in validation failures
+> enabling `EndpointSlice` feature gate will result in validation failures
 > during startup.
 
 #### Type NodePort, LoadBalancer, ClusterIP
 
-In a dualstack cluster Services can be:
+In a dual stack cluster, Services can be:
 1. Single Stack: service IP are allocated (or reserved, if provided by user)
    from the first or the second CIDR (if configured) entry as configured using
    `--service-cluster-ip-range` flag, in this scenario services will have a
    single allocated `ClusterIP` (also set in `ClusterIPs`).
 2. Dual Stack (optional): service IP is allocated (or reserved, if provided by
    user) from both the first and the second CIDR (if configured) entries. In
-   this scenario services will have a dual assigned `ClusterIPs`. If the
+   this scenario services will have one or two assigned `ClusterIPs`. If the
    cluster is not configured for dualstack then  the resulting service will be
```

Comment:
"dual stack" is missing a space there. (I wouldn't comment except that you fixed it a few lines up... maybe do a search for others? and move it to the "typos and acronyms" commit)

Reply:
It'll have to be an extra commit because too many conflicts.